packages/headless/src/controllers/breadcrumb-manager/headless-breadcrumb-manager.ts

```diff
@@ -144,3 +154,12 @@ export interface CategoryFacetBreadcrumb {
   path: CategoryFacetValue[];
   deselect: () => void;
 }
+
+const logClearBreadcrumbs = () =>
```

Comment:
This action should not be in the controller. 

I would put it in in `src/features/facets/generic/facet-generic-analytics-actions.ts`